openslides/mediafiles/access_permissions.py

```diff
@@ -18,3 +18,16 @@ def get_serializer_class(self, user=None):
         from .serializers import MediafileSerializer
 
         return MediafileSerializer
+
+    # def get_restricted_data(self, full_data, user):
```

Comment:
Do you want to use this code? How do you ensure, that only authorized users get private file data via autoupdate. Please have a look at #2357. This is under heavy development. :-(